### PR TITLE
Add Missouri tax rates for 2020 and 2021

### LINKED ISSUE #1482: Incorrect and missing MO income tax brackets
When comparing 2021 MO income taxes produced by PolicyEngine-US 0.176.0
versus TAXSIM35 10/24/22 version, we get many differences even for simple tax units.

One problem is that PolicyEngine seems to have incorrect 2019 MO income tax brackets
and they have not been updated for the years following 2019.

The official tax bracket/rate schedules for 2015 through 2021 are available at this [Missouri state government website](https://dor.mo.gov/forms/?category=3).

Also helpful in debugging MO income tax logic is this Missouri state government income tax [online
calculator](https://mytax.mo.gov/rptp/portal/home/indiv-income-tax-calculator).

Here is the test case showing the difference between the two models:
```
# PolicyEngine US test file derived from p21.its.csv and p21.ots.csv files

- name: Tax unit with taxsimid 5363 from p21.its.csv
  absolute_error_margin: 0.01
  period: 2021
  input:
    people:
      person1:
        is_tax_unit_head: 1
        is_tax_unit_spouse: 0
        is_tax_unit_dependent: 0
        age: 37
        employment_income: 160000
        ssi: 0
        state_supplement: 0
        wic: 0
      person2:
        is_tax_unit_head: 0
        is_tax_unit_spouse: 1
        is_tax_unit_dependent: 0
        age: 39
        employment_income: 160000
        ssi: 0
        state_supplement: 0
        wic: 0
    spm_units:
      spm_unit:
        members: [person1, person2]
        snap: 0
        tanf: 0
    tax_units:
      tax_unit:
        members: [person1, person2]
        premium_tax_credit: 0
    households:
      household:
        members: [person1, person2]
        state_code: MO
  output:
    mo_income_tax: 15550.33
```

The results for the above test case are:
```
================================= FAILURES ===================================
_______________________________ test session _________________________________
/Users/mrh/work/PolicyEngine-US/MO/p21-id5363.test.yaml:
  Test 'Tax unit with taxsimid 5363 from p21.its.csv':
    mo_income_tax@2021: [17244.] differs from 15550.330078125
                        with an absolute margin [1693.6699] > 0.01
========================= short test summary info ============================
FAILED p21-id5363.test.yaml::
============================ 1 failed in 0.94s ===============================
```

### PR DESCRIPTION
**Pull request by nmrodelo**
Fixes #1482

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Thanks, could you please:
* Run make format
* Add the changelog entry
* Add "Fixes #xx" to the initial description to link the issue (looks like the wrong one currently)
* Add the integration test that caused the failure reported in the issue

File: policyengine_us/parameters/gov/states/mo/tax/income/rates.yaml [hunk 1]
```diff
@@ -1,49 +1,68 @@
 description: The tax rates for individual residents in Missouri. Thresholds are <=.
 brackets:
   - rate:
-      2019-01-01: 0.015  
+      2019-01-01: 0.015
     threshold:
-      2019-01-01: 0
+      2019-01-01: 104
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/mo/tax/income/rates.yaml:6]: This isn't correct: The $104 is just for rounding purposes, the bracket starts at $0

**Comment by MaxGhenis** [comment]
@nmrodelo I made some fixes to the code but I think the tests are now incorrect, could you please update those?

**Comment by nmrodelo** [comment]
@MaxGhenis yep, will take a look later today

**Comment by martinholmer** [comment]
Nick @nmrodelo,  I notice you haven't had a chance to work on PR #1486 over the past two weeks.   Are you busy with other things?  Or are you stuck on a particular topic?  If the later, feel free to explain the problem you are facing here in the 1486 conversation.

**Comment by nmrodelo** [comment]
@martinholmer apologies, don't think i let anyone know i updated what Max asked for. went ahead and resolved the remaining merge conflicts, should be good as long as all the tests pass.

**Comment by martinholmer** [comment]
Nick @nmrodelo,  So PR #1486 is ready to merge as far as you can see?  Right?

**Comment by nmrodelo** [comment]
Yep!

**Comment by martinholmer** [comment]
Nick @nmrodelo said:
> Yep!

Thanks for you work on this!  I asked @MaxGhenis to re-review your latest additions.

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**